### ISSUE TITLE
Add the "-p" option to "wordup local:start"

### ISSUE THREAD

**Issue by stefano-b** [wordup-dev/wordup-cli#39]
I think it would be a nice to have the `-p` option even to `wordup local:start`, in a similar way to how it currently works on `wordup local:stop`.

Navigating to the project folder before starting the development environment could not be always the best option, while being able to start it from any location just by passing the project slug would be a very useful improvement.

An example would be to run `wordup list` to check the available projects, see their current status (installed/running), and directly type a command afterwards to start/stop one of them without the need to change the current working directory.

**Comment by shry** [comment]
Good idea. I will try to implement it as soon as possible.
